classifiers/sentdex/processor.py

```diff
@@ -0,0 +1,52 @@
+import warnings
+
+# from nltk.help import upenn_tagset
+from nltk import pos_tag
+from nltk.tokenize import word_tokenize
+
+from typing import Any, List
+
+class Processor:
+    def __init__(self, data: List[str], valid_word_tags: List[str], tag: str = None):
+        self.data: List[str] = data
+        self.valid_word_tags: List[str] = valid_word_tags
+        self.tag : str = tag
+        
+        self.documents: List[(str, str)] = []
+        self.tokenized_data: List[List[str]] = []
+
+    @property
+    def valid_word_tags(self) -> List[str]:
+        return self._valid_word_tags
+
+    @valid_word_tags.setter
+    def valid_word_tags(self, value: List[str]):
+        if not value:
+            warnings.warn('valid_word_tags will contain not tags')
+        # TODO: Raise AttributeError for invalid tags
```

Comment:
```TODO``` should be addressed.
Possibility for GitHub Actions to automate checks for ```TODO```/```FIXME``` in code.